core/cards.py

```
@@ -0,0 +1,128 @@
+    "barbarian hut": {"id": 74, "elixir": 5},
+    "rage": {"id": 75, "elixir": 1},
+    "zap": {"id": 76, "elixir": 2},
+    "giant snowball": {"id": 77, "elixir": 2},
+    "arrows": {"id": 78, "elixir": 3},
+    "tornado": {"id": 79, "elixir": 3},
+    "fireball": {"id": 80, "elixir": 4},
+    "poison": {"id": 81, "elixir": 4},
+    "freeze": {"id": 82, "elixir": 4},
+    "graveyard": {"id": 83, "elixir": 5},
+    "lightning": {"id": 84, "elixir": 6},
+    "fireball": {"id": 85, "elixir": 4},
```

Ruff (0.12.2)

Dictionary key literal `"fireball"` repeated

(F601)